x-pack/plugins/alerting/server/lib/alerting_event_logger/alerting_event_logger.ts

```diff
@@ -384,4 +402,15 @@ export function updateEvent(event: IEvent, opts: UpdateEventOpts) {
       total_search_duration_ms: metrics.totalSearchDurationMs ? metrics.totalSearchDurationMs : 0,
     };
   }
+
+  if (timings) {
+    event.kibana = event.kibana || {};
+    event.kibana.alert = event.kibana.alert || {};
+    event.kibana.alert.rule = event.kibana.alert.rule || {};
+    event.kibana.alert.rule.execution = event.kibana.alert.rule.execution || {};
+    event.kibana.alert.rule.execution.metrics = {
+      ...event.kibana.alert.rule.execution.metrics,
+      ...timings,
+    };
```

Comment:
lodash set method could save some lines here :) 
https://lodash.com/docs/4.17.15#set

Reply:
During some performance testing, it was noted that logging to the event log was an expensive operation for a large number of alerts and part of that was the usage of lodash, so I've been trying to remove usages when there are other alternatives :)